build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -276,7 +277,7 @@ export const handlers: Handler[] = [
 				json.homepage = homepage;
 			}
 
-			writeFile(file, JSON.stringify(sortPackageJson(json), undefined, 2) + newline);
+			writeFile(file, JSON.stringify(sortPackageJson(json), undefined, "\t") + newline);
```

Comment:
I think it would be good to use the updatePackageJsonFile util I wrote to implement this logic. It will need to be moved and exported though

Reply:
if you don't want to use that, use writeJsonSync at least to simplify this.

Reply:
It will also have to be moved to build-tools; otherwise build-tools would have to import from build-cli. I'd rather not add another export to build-tools since we're trying to retire it. I'll think about this more.

Reply:
I moved it into build-tools and used it in the relevant places. I did change the signature to make the transformer optional though.